src/App.tsx

```diff
@@ -1,11 +1,15 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
 
 function App() {
   return (
     <div className="App">
       <div className="title">Raman Rahinia</div>
-      <div className="info">I like to play video games like <a href="https://www.counter-strike.net/">CS2</a> or <a href="https://playvalorant.com/en-us/">VALORANT</a></div>
+      <div className="info">
+        I like to play video games like{" "}
+        <a href="https://www.counter-strike.net/">CS2</a> or{" "}
```

Comment:
the best practice for such cases is storing links as environment variables, let's use it